lib/config.js

```diff
@@ -0,0 +1,14 @@
+/*!
+ * Copyright (c) 2019 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
+const {config} = require('bedrock');
+
+const cfg = config['stats-storage-redis'] = {};
+
+cfg.history = {};
+// the length of history to maintain in redis, default 1 hour
+cfg.history.length = 60 * 60 * 1000;
```

Comment:
`ttl` or something like that might be better here.

I found the name for this option and the one below it a bit confusing because `purgeInterval` suggests all of history is removed (purged) on that interval, but clearly it can't mean that given that an hour of history is maintained (presumably when the "purge" happens). Maybe `pruneInterval` would be better than `purgeInterval`.

Reply:
done.